docs/modules/user-guide/partials/proc_defining-attributes.adoc

```diff
@@ -18,8 +21,30 @@ Use devfile attributes to configure various features and properties according to
 
 .Procedure
 
-
-. Define a custom attribute
+. Define attributes in a `component`
```

Comment:
```suggestion
. Define the `attributes` in `components`.
```

Reply:
Each entry in the components **array** is a single component - so I don't think it is correct to say 

>  Define the `attributes` in `components`

I could remove the ticks so that it says  "in a component" or "in components" - if we use ticks, then we should probably add the word "object" after  `component` or `components` , like we do with Kubernetes/OpenShift resources.

I will add a colon at the end of the statement, as it has a follow-up.